### ISSUE TITLE
Dropdown is preventing all keys when opened

### ISSUE THREAD

**Issue by driskull** [Esri/calcite-components#4614]
### Actual Behavior

When a dropdown is opened, you can't open the developer console because the component is calling event.preventDefault for all keys.

### Expected Behavior

A dev can open the console and only specific keys are prevented.

### Reproduction Sample

https://codepen.io/driskull/pen/yLvPjKw?editors=1000

### Reproduction Steps

1. Open the dropdown.
2. Try to open the console via keyboard
3. Notice that it won't open because keys are being prevented.

### Reproduction Version

latest

**Comment by macandcheese** [comment]
Agree- Probably just need to prevent esc, enter / return, space default behavior?
